src/nnet3/nnet-general-component.cc

```diff
@@ -991,8 +991,8 @@ void BackpropTruncationComponent::Init(int32 dim,
 void BackpropTruncationComponent::InitFromConfig(ConfigLine *cfl) {
   int32 dim = 0;
   bool ok = cfl->GetValue("dim", &dim);
-  BaseFloat clipping_threshold = 15.0;
-  BaseFloat zeroing_threshold = 2.0;
+  BaseFloat clipping_threshold = 30.0;
+  BaseFloat zeroing_threshold = 15.0;
```

Comment:
Larger values of these quantities are more dangerous, i.e. more likely to lead to instability.
I don't think it's sufficient to just test this on one setup, because it's the potential for divergence that this is supposed to guard against.  Have you done any other tests?